unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditor.cs

```diff
@@ -34,11 +36,20 @@ public class SkyboxEditorWindow : EditorWindow
         private List<RightPanelPins> rightPanelPins = new List<RightPanelPins>() { new RightPanelPins { part = SkyboxEditorToolsParts.BG_Layer, name = "Background Layer" } };
 
         private CopyFunctionality copyPasteObj;
-        private SkyboxElements skyboxElements;
+        private static SkyboxElements skyboxElements;
 
+        private static SkyboxConfiguration[] tConfigurations;
+        private static MaterialReferenceContainer materialReferenceContainer;
```

Comment:
Why do they need to be static?

Reply:
I made it static because in previous implementation there was a static call inside the Init() method. With the current implementation, that static reference was gone. So, the Init() method doesnt need to be static, therefore all these references dont need to be static.

Removed!

Reply:
Nice! thanks